XpringKit/ReliableSubmissionXpringClient.swift

```diff
@@ -48,7 +48,6 @@ extension ReliableSubmissionXpringClient: XpringClientDecorator {
       Thread.sleep(forTimeInterval: ledgerCloseTime)
 
       latestLedgerSequence = try getLatestValidatedLedgerSequence()
-      print("got latest: \(latestLedgerSequence)")
```

Comment:
Not a useful print?

Reply:
Yeah, this is just left over and I happened to notice and wanted to clean up.